website/docs/user_attributes.md

```diff
@@ -0,0 +1,166 @@
+|          **OIDC scope** | `openid`                                                                                        |
+|       **OIDC claim(s)** | `sub`                                                                                           |
+| **OIDC claim location** | <ul><li>ID token</li><li>Userinfo endpoint</li><li>Introspection endpoint</li></ul>             |
+|              **origin** | Check-in assigns this attribute on user registration                                            |
+|             **changes** | No                                                                                              |
+|        **multiplicity** | No                                                                                              |
+|        **availability** | Always                                                                                          |
+|             **example** | _4a58cb0338fb68060ba75e20e02a4844148f9c63a5cd9a7418dc5a9a6ab641cbaef73b4c65519286359f766c45bdbad36cc9189e75a03bd12f13526c34cab93e@eid-proxy.aai.grnet.gr_   |
+|               **notes** | Use **EGI ID** within your application as the unique-identifier key for the user                |
+|              **status** | Stable                                                                                          |
+
+<!-- markdownlint-enable line-length no-inline-html -->
+
+### 2. Person Identifier
+
+|          attribute name | Person Identifier                                                                   |
+| ----------------------: | :---------------------------------------------------------------------------------- |
+|         **description** | The user's person identifier                                                        |
+|   **SAML Attribute(s)** | `http://eidas.europa.eu/attributes/legalperson/PersonIdentifier` (PersonIdentifier) |
+|          **OIDC scope** | `profile`                                                                           |
+|       **OIDC claim(s)** | `person_identifier`                                                                 |
+| **OIDC claim location** | Userinfo endpoint                                                                   |
+|              **origin** | Provided by user's Identity Provider                                                |
+|             **changes** | Yes                                                                                 |
+|        **multiplicity** | Single-valued                                                                       |
+|        **availability** | Always                                                                              |
+|             **example** | _CA/CA/UniqueId1_                                                                   |
+|               **notes** | -                                                                                   |
```

Comment:
notes: Any service that consumes assertions of identity must assume that the Unique Identifier presented for a particular person may change over time e.g. where the user's digital identity is replaced or repaired. This should be handled by a consuming service using the same matching process as used when an identity is first encountered utilising the Minimum Data Set to identify the Principal within the service.